packages/react-strict-dom/src/dom/runtime.js

```diff
@@ -177,3 +177,5 @@ export const defaultStyles = {
   u: u as typeof u,
   ul: ul as typeof ul
 };
+
+export const resolveStyle = stylex.props;
```

Comment:
This can changed to a bespoke styleq configuration for some minor runtime optimization (`disableMix:true`) and to avoid some bundle bloat from importing the StyleX runtime.